roles/rsyslog/defaults/main.yml

```diff
@@ -221,28 +185,6 @@ rsyslog_tcp_port: '514'
 # The incoming TCP TLS port used for remote logging.
 rsyslog_tcp_tls_port: '6514'
 
-# rsyslog_allow
```

Comment:
Did you mean to get rid of these?

Reply:
Yes, these rsyslog_.*allow variables are from debops and they are not used in the LSR logging for now.